react/features/transcribing/components/AbstractTranscribingLabel.js

```diff
@@ -5,8 +5,29 @@
  */
 export type Props = {
 
+    /**
+     * True if the label needs to be rendered, false otherwise.
+     */
+    _showTranscribingLabel: boolean,
```

Comment:
Since this is the transcribing label it seems a bit redundant to mention it again, doesn't it? Maybe just call it `_show` ?

Reply:
I renamed it to _showLabel as I feel _show to be too generic (e.g. it could even mean to show transcribing)